pkg/profilecreator/profilecreator.go

```diff
@@ -312,6 +307,34 @@ func (ghwHandler GHWHandler) GetReservedAndIsolatedCPUs(reservedCPUCount int, sp
 
 }
 
+// getCPUsSequentially returns Reserved and Isolated CPUs sequentially
+func (ghwHandler GHWHandler) getCPUsSequentially(reservedCPUCount int, topologyInfoNodes []*topology.Node) (string, string, error) {
+	totalCPUSet := cpuset.NewBuilder()
+	reservedCPUSet := cpuset.NewBuilder()
+	max := reservedCPUCount
+	htEnabled, err := ghwHandler.isHyperthreadingEnabled()
+	if err != nil {
+		return "", "", fmt.Errorf("Error determining if Hyperthreading is enabled or not: %v", err)
+	}
+	if max%2 != 0 && htEnabled {
+		return "", "", fmt.Errorf("Can't allocatable odd number of CPUs from a NUMA Node")
+	}
+	for _, node := range topologyInfoNodes {
```

Comment:
if we add the `totalCPUSetFromTopology` helper I suggest below, we can reason about the expanded cpu list here and probably use a clearer logic. In this case, if I'm not mistakens, it boils down to just pick the first `reservedCPUSet` items in the sorted slice. Right?

Reply:
no pointing in changing the logic if we don't _also_ change the other helper (see comment below), so scratch this.

Reply:
Ack